src/aasthaJJun20/InvertTheCaseOfArrayElements.java

```diff
@@ -0,0 +1,43 @@
+package aasthaJJun20;
+
+/*WAP to convert case in all elements of array. 
+Input : 
+String[] array = {"AaVruti", "NiKhil", "AdiTi", "MAhesh", "ArchaNA", "jaGAdeesh"};
+
+Output:
+array = {"aAvRUTI", "nIkHIL","aDItI", "maHESH","aRCHAna","JAgaDEESH"};*/
+
+public class InvertTheCaseOfArrayElements {
+	//Below method displays the array elements on console
+	void diplayArrayElements(String[] input) {
+		System.out.print("{  ");
+		for (int index = 0; index < input.length; index++)
+			System.out.print(input[index] + "  ");
+		System.out.print("}");
+	}
+	
+	//Below method converts the case of every character of every array element
+	void invertCase(String[] input) {
+		System.out.print("Input array : ");
+		diplayArrayElements(input);
```

Comment:
Arrays.toString(input) Just need to write this line. No need to of diplayArrayElements() method.

Reply:
Got it Krishna!! Thank you!!